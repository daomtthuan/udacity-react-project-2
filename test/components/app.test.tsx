import App from '~components/app';

import { render } from '@testing-library/react';

describe('Renders correctly', () => {
  test(`${App.name} should render as expected`, () => {
    const component = render(<App />);

    expect(component.baseElement).toMatchSnapshot();
  });
});
