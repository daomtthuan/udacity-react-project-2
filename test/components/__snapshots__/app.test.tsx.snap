// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly App should render as expected 1`] = `
<body>
  <div />
</body>
`;
